Gemfile

```diff
@@ -26,14 +26,15 @@ gem 'simple_calendar', :git => 'https://github.com/harshared/simple_calendar.git
 
 ## Verify Rails 5 eliminates need for this gem with MongoDB
 gem 'database_cleaner',       '~> 1.7'
+gem 'pry'
 
 #######################################################
 
 #######################################################
 # Local components/engines
 #######################################################
 gem 'acapi',              git: "https://github.com/ideacrew/acapi.git", branch: 'amqp_proc_title'
-gem 'aca_entities',       git: 'https://github.com/ideacrew/aca_entities.git', branch: 'release_0.7.0'
+gem 'aca_entities',       git: 'https://github.com/ideacrew/aca_entities.git', branch: 'mcr_mapping'
```

Comment:
should this be pointing to a feature branch?

Reply:
Yes! will update this before merging

Reply:
updated